resources/language/English/strings.po

```diff
@@ -180,3 +180,7 @@ msgstr ""
 msgctxt "#30048"
 msgid "WEEK {0}"
 msgstr ""
+
+msgctxt "#30049"
+msgid "Use InputStream Adaptive"
```

Comment:
@emilsvennesson Ahhh, I didn't realize that. So, this uses InputStream.Adaptive but InputStream.Adapative does not yet support adaptive bitrates.

If that's the case, I agree with you to keep the wording as-is. :-)

---Alex